server_addon/houdini/client/ayon_houdini/plugins/load/load_hda.py

```diff
@@ -1,12 +1,33 @@
 # -*- coding: utf-8 -*-
 import os
-from ayon_core.pipeline import get_representation_path
+import hou
+from ayon_core.pipeline import (
+    get_representation_path,
+    AVALON_CONTAINER_ID
+)
 from ayon_core.pipeline.load import LoadError
 from ayon_houdini.api import (
+    lib,
     pipeline,
     plugin
 )
 
+def get_avalon_container():
```

Comment:
I feel like this logic makes more sense in `lib` when taken out of `containerise` into a `get_or_create_avalon_container()` function that `containerise` then also ends up using. That way we ensure the logic remains synced and avoids custom logic here.

Reply:
agreed. 
Also, for reference this logic is found in load image
https://github.com/ynput/ayon-core/blob/506c9975af1237635827fdc45459ab57c4e2fed1/server_addon/houdini/client/ayon_houdini/plugins/load/load_image.py#L24-L35

Reply:
ehm, should it exist in lib or pipeline

Reply:
I'd say `pipeline` :)

Reply:
thank you https://github.com/ynput/ayon-core/pull/456/commits/336d38b21835f02bd04b5a2a4e317d07e5bb94b1